lib/logstash/filters/kubernetes_metadata.rb

```diff
@@ -0,0 +1,223 @@
+# encoding: utf-8
+require "logstash/filters/base"
+require "logstash/namespace"
+require "lru_redux"
+require "rest-client"
+require "uri"
+require "logstash/json"
+
+# Process the metadata include in the file names of kubernetes log files
```

Comment:
Can you add more information here? This will be converted to user facing documentaion.